include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -161,10 +161,10 @@ struct __get_first_range_type
 };
 
 template <typename _Range, typename... _Ranges>
-constexpr _Range
-__get_first_range(_Range&& __rng, _Ranges&&...)
+auto
```

Comment:
I would not recommend to return `auto` even if we already require C++17 to be able to use ranges API. The problem is if this header explicitly or implicitly would occur in other headers that can work with C++11 that would break the code until C++14 because this function will not pass the basic syntax check for C++11 with `auto` return type.

Reply:
I will fix it in the next commit.

Reply:
Yes, that header file is "C++11 compliance", at least...